src/host/trace.c

```diff
@@ -42,7 +42,7 @@
 /* testbench trace definition */
 
 /* enable trace by default in testbench */
-static int test_bench_trace = 1;
+int test_bench_trace = 1;
 int num_trace_classes;
 
 /* set up trace class identifier table based on SOF trace header file */
```

Comment:
Please do eradicate the `setup_trace_table` abomination.

Reply:
And any _no longer needed functions/macros_ from `host/trace.c` and `host/trace.h`.

Reply:
It is required for host testbenches.

Reply:
`int test_bench_trace = 1;` yes
but not `setup_trace_table` function.

Reply:
That's right. I've updated it.

Reply:
Great, it feels so **pure** now.